plat/allwinner/sun50i_h6/sunxi_power.c

```diff
@@ -119,3 +120,24 @@ int sunxi_pmic_setup(void)
 
 	return 0;
 }
+
+void __dead2 sunxi_power_down(void)
+{
+	uint8_t val;
+
+	switch (pmic) {
+	case AXP805:
```

Comment:
I think we need to initialise the I2C driver here again. The non-secure OS could have configured this entirely differently or even disabled I2C (especially the platform part), so we can't rely on our initialisation at boot time still being valid. This makes the initialisation at boot time somewhat questionable, but it's still useful for detecting the PMIC, selecting and reporting it.

Can be easily fixed with a follow up patch.